pennylane_lightning/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.22.0-dev16"
+__version__ = "0.22.0-dev17"
```

Comment:
We can hold off on merging this as the new tag will push later today/early tomorrow.